File: client-java/instrumentation/src/test/java/com/foo/somedifferentpackage/examples/strings/StringCallsImp.java
package com.foo.somedifferentpackage.examples.strings;

import org.evomaster.clientJava.instrumentation.example.strings.StringCalls;

public class StringCallsImp implements StringCalls {
    @Override
    public boolean callEquals(String a, Object b) {
        if(a.equals(b))
            return true;
        return false;
    }

    @Override
    public boolean callEqualsIgnoreCase(String a, String b) {
        if(a.equalsIgnoreCase(b))
            return true;
        return false;
    }

    @Override
    public boolean callStartsWith(String a, String b) {
        if(a.startsWith(b))
            return true;
        return false;
    }

    @Override
    public boolean callStartsWith(String a, String b, int toffset) {
        if(a.startsWith(b, toffset))
            return true;
        return false;
    }

    @Override
    public boolean callEndsWith(String a, String b) {
        if(a.endsWith(b))
            return true;
        return false;
    }

    @Override
    public boolean callIsEmpty(String a) {
        if(a.isEmpty())
            return true;
        return false;
    }

    @Override
    public boolean callContentEquals(String a, CharSequence cs) {
        if(a.contentEquals(cs))
            return true;
        return false;
    }

    @Override
    public boolean callContentEquals(String a, StringBuffer sb) {
        if(a.contentEquals(sb))
            return true;
        return false;
    }

    @Override
    public boolean callContains(String a, CharSequence cs) {
        if(a.contains(cs))
            return true;
        return false;
    }
}

File: outdated/outdated-client-dotnet/test/EvoMaster.Instrumentation.Examples/Strings/StringCallsImp.cs
using System;
using EvoMaster.Instrumentation.Examples.strings;

namespace EvoMaster.Instrumentation.Examples.Strings{
    public class StringCallsImp : IStringCalls{
        public bool callEquals(string a, object b){
            if(a.Equals(b))
                return true;
            return false;
        }

        public bool callEqualsStringComparison(string a, string b, StringComparison comparison){
            if(a.Equals(b, comparison))
                return true;
            return false;
        }

        public bool callStartsWith(string a, string b){
            if(a.StartsWith(b))
                return true;
            return false;
        }

        public bool callStartsWithStringComparison(string a, string b, StringComparison comparison){
            if(a.StartsWith(b, comparison))
                return true;
            return false;
        }

        public bool callEndsWith(string a, string b){
            if(a.EndsWith(b))
                return true;
            return false;
        }

        public bool callEndsWithStringComparison(string a, string b, StringComparison comparison){
            if(a.EndsWith(b, comparison))
                return true;
            return false;
        }

        public bool callContains(string a, string cs){
            if(a.Contains(cs))
                return true;
            return false;
        }

        public bool callContainsStringComparison(string a, string cs, StringComparison comparison){
            if(a.Contains(cs, comparison))
                return true;
            return false;
        }
    }
}